### ISSUE TITLE
Version compatibility tables should treat Edge 12 as version 1

### ISSUE THREAD

**Issue by MarkMaldaba** [mdn/browser-compat-data#9070]
The version compatibility tables will show 'Yes' if a browser has always supported a feature, otherwise it will show the version in which the feature was introduced.

However, because Microsoft Edge started at version 12, it is being treated differently, and in a lot of cases the 'Edge' column will show `12` rather than `Yes` which can cause confusion, as it implies that there are versions of Edge that did not support the feature.

Take the page on status codes, for example: [/docs/Web/HTTP/Status](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#browser_compatibility).  Anyone unfamiliar with the situation would think that all browsers have supported the majority of these codes, except for Edge which only introduced them in version 12.  Even for people who understand the situation, there is a bit of extra parsing to realise that there is no compatibility concern introduced by that column.

tl;dr Replace all instances of Edge: 12 with Edge: Yes.

**Comment by vinyldarkscratch** [comment]
There's been a lot of confusion about what "Yes" means in the compatibility tables.  Actually, "Yes" does not mean that the feature was always supported in every version -- what it means is "it was supported at some point, we just don't know exactly when".  We're working to remove all these "Yes" values from the data, see #6369.

I agree that it's confusing for developers that EdgeHTML started at version 12, certainly posing a bit of an additional challenge if consumers aren't familiar that Edge is pretty much a direct continuation off of Internet Explorer.  Alas, Microsoft requested we keep Internet Explorer and Edge separate.
